draft-packages/popover/KaizenDraft/Popover/AppearanceAnim.tsx

```diff
@@ -4,7 +4,7 @@ import { useDebouncedCallback } from "use-debounce"
 import styles from "./AppearanceAnim.scss"
 
 type Props = {
-  children: React.ReactNode
+  children?: React.ReactNode
```

Comment:
Did this definitely need to happen? I can't see any usages that aren't using children and it seems like it probably should be a required prop.